src/compiler/diagnosticMessages.json

```diff
@@ -2872,26 +2872,30 @@
         "category": "Error",
         "code": 7015
     },
-    "Property '{0}' implicitly has type 'any', because its 'set' accessor lacks a type annotation.": {
+    "Property '{0}' implicitly has type 'any', because its set accessor lacks a parameter type annotation.": {
         "category": "Error",
         "code": 7016
     },
-    "Index signature of object type implicitly has an 'any' type.": {
+    "Property '{0}' implicitly has type 'any', because its get accessor lacks a return type annotation.": {

```

Comment:
In fact, why not just use `{1}` to add a get/set parameter and re-use error 7016?